site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -193,11 +193,11 @@ describe('Test cases revolving around polls functionality', () => {
 
         // make a poll
         cy.contains('New Poll').click();
-        cy.get('#poll-name').type('Poll Cypress Test');
-        cy.get('#poll-question').type('# Question goes here...?');
-        cy.get('#poll-date').clear({ force: true });
+        cy.get('[data-testid="poll-name"]').type('Poll Cypress Test');
+        cy.get('[data-testid="poll-question"]').type('# Question goes here...?');
+        cy.get('[data-testid="poll-date"]').clear({ force: true });
```

Comment:
is it necessary to ` force:true` clear, can we do it without it ?
same for other place too

Reply:
This area of the Poll cypress tests should be left functionally untouched by this PR. I only changed to selector for consistency.